src/Chapter1.hs

```diff
@@ -478,7 +478,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Integer -> Integer --Integral a => a -> a didn't work for some reason
+lastDigit n = mod n 10
```

Comment:
Your implementation is almost correct :ok: 
Unfortunately, it returns negative numbers on negative inputs because of how `mod` works. Sometimes corner cases can be tricky to spot and fix...